htmresearch/regions/TMRegion.py

```diff
@@ -117,22 +116,32 @@ def getSpec(cls):
         ),
         outputs=dict(
             bottomUpOut=dict(
-              description="""The primary output of the temporal memory.""",
+              description="""The default output of TMRegion. By default this
+                   outputs the active cells. You can change this dynamically
+                   using the defaultOutputType parameter.
+                   """,
```

Comment:
@vitaly-krugl Thanks - I think my recent push fixed it.  However I do now see there are other places in the file where the same mistake is made. I can take care of that in a later PR. Sorry, my reply doesn't rhyme! 😄